indigo/bulk_creator.py

```diff
@@ -123,9 +120,8 @@ def get_datatable_gsheets(self, spreadsheet_id, sheet_name):
         index (tab index).
         """
         try:
-            result = self.gsheets_client\
-                .spreadsheets()\
-                .values().get(spreadsheetId=spreadsheet_id, range=sheet_name)\
+            result = list(self.gsheets_client\
```

Comment:
Was this an automatic correction? I don't think it's correct. The call to `values()` isn't a dictionary call. I think the whole thing can remain unchanged.

Reply:
Yes, this was an automatic one, as were the other `list()` changes